frontend/src/pages/GraphicalSelector/CourseInfo/GraphicalPlannerButton.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { PlusOutlined, StopOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import axios from 'axios';
+import { Course, UnselectCourses } from 'types/api';
+import { PlannerCourse } from 'types/planner';
+import prepareUserPayload from 'utils/prepareUserPayload';
+import type { RootState } from 'config/store';
+import { addToUnplanned, removeCourses } from 'reducers/plannerSlice';
+
+interface PlannerButtonCodeProps {
+  course: Course;
+  onChange?: () => void;
+}
+
+const PlannerButtonCode: FunctionComponent<PlannerButtonCodeProps> = ({ course, onChange }) => {
```

Comment:
Pretty sure we can reused the AddToPlanner button?

Reply:
we couldn't but now we are using a refactored common one!